roles/pulp/tasks/install.yml

```diff
@@ -191,6 +180,51 @@
         line: "include-system-site-packages = true"
       when: pulp_use_system_wide_pkgs | bool
 
+    - name: Create requirements.in file to check pulpcore/plugin compatibility
+      template:
+        src: requirements.in.j2
+        dest: "{{ pulp_install_dir }}/requirements.in"
+      when: pulp_source_dir is undefined
+
+    - name: Install pip-tools before running pip-comile to check version compatibility
+      pip:
+        name: pip-tools
+        state: present
+        virtualenv: '{{ pulp_install_dir }}'
+        virtualenv_command: '{{ pulp_python_interpreter }} -m venv'
+
+    - name: Run pip-compile to check pulpcore/plugin compatibility
+      command: '{{ pulp_install_dir }}/bin/pip-compile'
+      args:
+        chdir: '{{ pulp_install_dir }}'
+      register: compatibility
+      ignore_errors: true
+      environment:
+        LC_ALL: en_US.UTF-8
+        LANG: en_US.UTF-8
```

Comment:
I'm not sure if we can assume this language pack is installed, particularly on Fedora 31.

Reply:
I added these environment variables because pip-compile was throwing an error about it in our CI. I got the idea from here: https://stackoverflow.com/questions/36651680/click-will-abort-further-execution-because-python-3-was-configured-to-use-ascii